src/index.js

```diff
@@ -1,86 +1,104 @@
+/* eslint-disable radix */
+/* eslint-disable import/no-cycle */
+
```

Comment:
Linters help us to write clean code so disabling them means we are doing ourselves a disservice. 

- [ ] Please kindly enable them and fix them instead.

Reply:
Done